common/defs-flags.go

```diff
@@ -1,16 +1,16 @@
 package common
 const (
 	// Shared flag definitions
-	FlagEnvDefinition = "Name of the environment to read environmental variables from."
+	FlagEnvDefinition = "Environment to read environmental variables from."
```

Comment:
This sentence is confusing, let's get rid of the word environmental, it doesn't apply here.